.platform.app.yaml

```diff
@@ -26,14 +26,19 @@ hooks:
       # Install and compile GitBook
       npm install
       npm run build
+
+      # Build example configuration files from the Registry so they can be pulled into docs.
+      node src/scripts/updateExampleConfigFiles.js
+
       gitbook install
       gitbook build
 
       # Gitbook doesn't allow for a custom favicon except through a theme plugin. So Plan B.
       cp src/images/favicon.ico _book/gitbook/images/favicon.ico
 
-      # Build example configuration files from the Registry.
-      node _book/scripts/updateExampleConfigFiles.js
+      # Move example configuration file API points to _book.
+      mv src/registry/images/examples _book/registry/images
+      mv src/registry/images/registry.yaml _book/registry/images/
```

Comment:
Wouldn't they get pulled over automatically if they're built in src before the book is built?

Reply:
The git commit stuff is a red herring.  That's all irrelevant to this move command (which should maybe be a recursive copy?)  The issue here is that the files are built above, but not copied over by GitBook automatically so we have to do it manually.  That the directory is gitignored is beside the point.